### PR TITLE
[EN] update containerization

### PR DESCRIPTION
**Pull request by NitishRawat123**
easier way to define containersiation.

### Checklist before opening this PR (put `x` in the checkboxes)
- [ ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/en/containerization.md [hunk 1]
```diff
@@ -5,10 +5,7 @@ category: Technology
 tags: ["application", "", ""]
 ---
 
-Containerization is the process of bundling an application and its dependencies into a container image. 
-The container build process requires adherence to the [Open Container Initiative](https://opencontainers.org) (OCI) standard. 
-As long as the output is a container image that adheres to this standard, which containerization tool is used doesn't matter.
-
+Containerization is the packaging of software code with just the operating system (OS) libraries and dependencies required to run the code to create a single lightweight executable—called a container, that runs consistently on any infrastructure. More portable and resource-efficient than virtual machines (VMs). They have become the de facto compute units of modern cloud-native applications.
```
Comment (iamNoah1) [inline content/en/containerization.md:8]:
```suggestion
Containerization is the process of packaging of application code including libraries and dependencies required to run the code into a single lightweight executable—called [container image](/container-image/). 
```

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
Thanks @NitishRawat123 for your contribution. I agree that your approach is more simple and straight forward to understand. I added the link to container image and removed information that is already present in the pages of the other terms.

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Review by nate-double-u — APPROVED** [review]
Thanks for this @NitishRawat123 

/approve

**Pull request merged**